### ISSUE TITLE
Installing BlackBerry-Dynamics-for-React-Native-Base throws an error

### ISSUE THREAD

**Issue by ign699** [blackberry/BlackBerry-Dynamics-React-Native-SDK#5]
Hi,
I've found an issue when trying to install `BlackBerry-Dynamics-for-React-Native-Base`. `npm i <path>/modules/BlackBerry-Dynamics-for-React-Native-Base` throws an `Error: ENOENT: no such file or directory, copyfile` error when trying to copy `BlackBerry-Dynamics-for-React-Native-Base/scripts/custom_rn_rename/index.js` into node_modules.

**Comment by JeffinWithYa** [comment]
Hi @ign699 ,

Are you installing the module in your own app or using one of the sample apps? Can you try installing the base module to one of our samples and see if you can replicate the issue?

**Comment by ign699** [comment]
Yes, it worked when installing inside the SampleApp but I couldn't get it to work when trying to install it on a standalone app.

**Comment by JeffinWithYa** [comment]
@ign699 can you confirm for us what version of React-Native you are using and also what version of the BlackBerry Dynamics SDK you are using (iOS and Android)?
